assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -43,6 +43,7 @@ inputs:
     enum:
       - Huggingface
       - MMLab
+      - pyfunc_llava
```

Comment:
can we have just Llava?

Reply:
@gauravrajguru was saying "Microsoft_Internal".
@novaturient95 is saying "llava".
@rdondera-microsoft is saying "pyfunc_llava".
None of these seem good names for a framework but "pyfunc_llava" tells you the flavor is pyfunc and it's somehow related to llava. Other thoughts?